pygeoapi/provider/csv_.py

```diff
@@ -149,11 +167,11 @@ def query(self, startindex=0, limit=10, resulttype='results',
         :param datetime: temporal (datestamp or extent)
         :param properties: list of tuples (name, value)
         :param sortby: list of dicts (property, order)
+        :param filter_expression: string of a filter expression
 
         :returns: dict of GeoJSON FeatureCollection
         """
-
-        return self._load(startindex, limit, resulttype)
+        return self._load(startindex, limit, resulttype, filter_expression=filter_expression) # noqa
```

Comment:
same above, you should format the code rather than add noqa

Reply:
done